pkg/sfu/streamtracker.go

```diff
@@ -39,6 +39,8 @@ type StreamTrackerParams struct {
 	BitrateReportInterval time.Duration
 
 	Logger logger.Logger
+
+	Layer int32
```

Comment:
Is this param used anywhere? That's the only change I see in this file.

Reply:
Unused now, will remove it